src/components/inputs/number-input/number-input.spec.js

```diff
@@ -75,21 +75,6 @@ describe('NumberInput', () => {
     expect(container.querySelector('input')).toHaveAttribute('type', 'number');
   });
 
-  it('should have pristine style when untouched', () => {
-    const { container } = render(<NumberInput {...baseProps} />);
-    expect(container.querySelector('input')).toHaveClass('pristine');
```

Comment:
Those tests are not necessary, since we have VRT.
Also, this doesn't work anymore as-is, because the class names from emotion are different.